README.md

```diff
@@ -1,2 +1,5 @@
-# wp-cloud-mgmt-console-plugin
-WordPress plugin that links your installation to the WordPress Cloud Management Console and Support. This Console provides a great user experience for launching and managing all your production WordPress deployments.
+# WordPress Cloud Management Console Plugin
+This plugin links your installation to the WordPress Cloud Management Console and Support. This Console provides a great user experience for launching and managing all your production WordPress deployments.
+
+View in the Admin panel:
+![View](https://raw.githubusercontent.com/bitnami-labs/wp-cloud-mgmt-console-plugin/master/images/demo.png)
```

Comment:
I'd include a couple of instructions to install the plugin (since it's not a plugin published at https://wordpress.org/plugins/)